tests/utils_tests/test_dateparse.py

```diff
@@ -116,6 +116,12 @@ def test_negative(self):
             ('-15:30', timedelta(minutes=-15, seconds=30)),
             ('-1:15:30', timedelta(hours=-1, minutes=15, seconds=30)),
             ('-30.1', timedelta(seconds=-30, milliseconds=-100)),
+            ('-00:01:01', timedelta(minutes=1, seconds=1)),
+            ('-01:01', timedelta(seconds=-59)),
```

Comment:
I believe the reporter suggested that this should be -61 seconds. If this is correct, it seems that existing cases like '-15:30' also need to be changed.

Reply:
I misunderstood the suggestion, I'll fix them.